### DISCUSSION TITLE
cornerPoints

### DISCUSSION THREAD

**Discussion by angelLain** [capawesome-team/capacitor-mlkit#69] [Q&A] [answered]
as I can draw the cornerPoints in my html, try canvas but the dots don't match the size of my screen

**Comment by robingenz** [comment]
You need to scale the coordinates, see https://github.com/robingenz/capacitor-mlkit-plugin-demo/blob/main/src/app/modules/barcode-scanning/barcode-scanning-modal.component.ts.

**Reply by angelLain** [reply]
thanks for your quick response, but when I see the repository of the link I only see that they scale a square to omit the codes that are outside that square, but my problem is that the cornerPoints return me coordinates of up to 1000px but the screen of my cell phone only reaches up to 700 which makes it impossible to draw a cavas for its representation

**Reply by robingenz** [reply]
Yes, you need to use `window.devicePixelRatio` like in the example.

**Reply by angelLain** [reply, marked as answer]
Thank you very much, it was just to divide the coordinates with the window.devicePixelRatio :3
